scripts/init.sh

```diff
@@ -2,7 +2,16 @@
 
 set -e
 
+echo "Setup git hooks"
+git config core.hooksPath hooks/
+
+echo "Update submodules"
 git submodule update --init --recursive || true
+
+echo "Installing dev tools"
+command -v sszgen || go install github.com/ferranbt/fastssz/sszgen@latest
```

Comment:
This `init.sh` script only needs to be called once when cloning the repo, so we don't to test whether `sszgen` exists.

```suggestion
go install github.com/ferranbt/fastssz/sszgen@latest
```